<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellForm</class>
 <widget class="QWidget" name="SellForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">*{
color:white;
background-color:rgb(54,54,54);
font-size:12;
}
QPushButton:hover{
border:1px solid white;
}
QPushButton:pressed{
border:2px solid orange;
}
QFrame#SearchFrame{
border:1px solid orange;
border-radius:10px;
font:14px;
}
QLineEdit#SearchLineEdit{
border:none;
border-bottom:2px solid grey;
}
QPushButton#AddToCartBtn{
background-color:green;
border-radius:10px;
font:14px;
}
QPushButton#AddToCartBtn:hover{
background-color:#00d500;
}
QPushButton#SearchBtn{
background-color:green;
border-radius:10px;
font:14px;
}
QPushButton#SearchBtn:hover{
background-color:#00d500;
}
QWidget{
padding:5px;}
QWidget#PaymentsWidget QPushButton{
background-color:green;
font:14px;
border-radius:10px;
}
QWidget#PaymentsWidget QPushButton:hover{
background-color:#00d500;
}
QLabel#Totalslabel{
font:18px;}
QToolTip{
background-color:black;
border:1px solid grey;
color:white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="SearchFrame">
          <property name="minimumSize">
           <size>
            <width>530</width>
            <height>70</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="searchLE">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="maxLength">
              <number>100</number>
             </property>
             <property name="placeholderText">
              <string>Enter Item Name</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Quantity :</string>
             </property>
             <property name="buddy">
              <cstring>qtySpinbox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="qtySpinbox">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border:1px solid grey;</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AddToCartBtn">
             <property name="minimumSize">
              <size>
               <width>135</width>
               <height>40</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Add item To Cart</string>
             </property>
             <property name="text">
              <string>Add To Cart (F5)</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/icons/add_shopping_cart_48px_gray.png</normaloff>:/icons/icons/add_shopping_cart_48px_gray.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SearchBtn">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/icons/search_database_48px_grey.png</normaloff>:/icons/icons/search_database_48px_grey.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:0px;</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>40</number>
        </attribute>
        <column>
         <property name="text">
          <string>Product Name</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="background">
          <color>
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Unit Price</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="background">
          <color>
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quantity</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="background">
          <color>
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Action</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="background">
          <color>
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sub-Total</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="background">
          <color>
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>70</red>
            <green>70</green>
            <blue>70</blue>
           </color>
          </brush>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(191, 191, 191);
text-decoration: underline;</string>
            </property>
            <property name="text">
             <string>TOTALS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="totalsLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25pt &quot;MS Shell Dlg 2&quot;;
color: rgb(226, 0, 0);</string>
            </property>
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="PaymentsWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>340</width>
            <height>200</height>
           </size>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Cash Payment OR </string>
             </property>
             <property name="buddy">
              <cstring>payoptionBtn</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="payoptionBtn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Choose other payment (F6)</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/icons/coins_grey.png</normaloff>:/icons/icons/coins_grey.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Amount Paid</string>
             </property>
             <property name="buddy">
              <cstring>cashamountLE</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Change</string>
             </property>
             <property name="buddy">
              <cstring>changeamountLE</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="changeamountLE">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Balance</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="cashpaymentBtn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Complete Cash Transaction (F7)</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/icons/bill_64px_grey.png</normaloff>:/icons/icons/bill_64px_grey.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="cashamountLE">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>Enter Amount Paid</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchLE</tabstop>
  <tabstop>qtySpinbox</tabstop>
  <tabstop>AddToCartBtn</tabstop>
  <tabstop>SearchBtn</tabstop>
  <tabstop>payoptionBtn</tabstop>
  <tabstop>cashamountLE</tabstop>
  <tabstop>changeamountLE</tabstop>
  <tabstop>cashpaymentBtn</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SearchBtn</sender>
   <signal>clicked()</signal>
   <receiver>SellForm</receiver>
   <slot>open_searchWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_searchWindow()</slot>
 </slots>
</ui>
